sync/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Library for leveraging the power of Sync"""
-__version__ = "0.0.2"
+__version__ = "0.0.3"
```

Comment:
I think we'll want to force-push v0.0.3 - jumped the gun on pushing that up. Probably ok as long as there aren't any big interface changes.

Reply:
nm. let's make this 0.0.4. I think at least Pete is using v0.0.3 right now... don't want to pull the rug out from anyone...